ksql-rest-app/src/test/java/io/confluent/ksql/api/ApiTest.java

```diff
@@ -296,16 +298,19 @@ public void shouldRejectWhenInternalErrorInProcessingPullQuery() throws Exceptio
   }
 
   @Test
+  @CoreApiTest
   public void shouldCloseQuery() throws Exception {
 
     // Create a write stream to capture the incomplete response
     ReceiveStream writeStream = new ReceiveStream(vertx);
 
     VertxCompletableFuture<HttpResponse<Void>> responseFuture = new VertxCompletableFuture<>();
     // Make the request to stream a query
-    client.post("/query-stream")
-        .as(BodyCodec.pipe(writeStream))
-        .sendJsonObject(DEFAULT_PUSH_QUERY_REQUEST_BODY, responseFuture);
+    streamRequest("/query-stream", (request) ->
```

Comment:
Maybe just call it sendRequest()? As it can be used for sending arbitrary requests.

Reply:
Ah interesting, I had named it `streamRequest()` not in reference to the endpoint (the same helper method is also used for the `/inserts-stream` endpoint in one of the tests) but because of how the request sender opens a write stream to stream back the response. As you've pointed out, though, the name is a bit confusing and the request sender could be more generic so I've renamed this to `sendRequest()` as you've suggested.

My original concern with this was that overloading `sendRequest()` could be confusing but I think it's fine for now.